views/data.html

```diff
@@ -0,0 +1,76 @@
+{% extends "layouts/one-column.html" %}
+
+{% block breadcrumb %}
+  <li><a href="/">Home</a></li>
+  <li>{{ title | truncate(35) }}</li>
+{% endblock %}
+
+{% block main %}
+  <article>
+    <section>
+      <h2>Overview</h2>
+      <p>
+        The entire OpenTrials database is saved monthly automatically. This
+        is what we use to power our API, and contains everything that you can
+        see on this website. Having the entire database in your local machine
+        allows you to analyze the data using your favorite tool, or even create
+        your own copy of the OpenTrials API.
+      </p>
+      <p>
+        As with any usage of OpenTrials's data, make sure you read and
+        understand our <a href="/terms-of-use">Terms of use</a>.
+      </p>
+    </section>
+    <section>
+      <h2>Download</h2>
+      {% if dumps.length > 0 %}
+        <p>
+          Below are the latest available dumps of our database. Their
+          file names contain the date at which the dump was produced:
+        </p>
+        <ul>
+          {% for dump in dumps | sort(true, false, 'label') %}
+          <li>
+            <a href="{{ dump.url }}" rel="external">
+                {{ dump.label }}
+            </a> ({{ dump.size | formatBytes }})
+          </li>
+          {% endfor %}
+        </ul>
+      {% else %}
+        <p>
+          Unfortunately, something went wrong and the list of dumps could
+          not be obtained. Please try again later. If the issue remains,
+          please <a href="/flag-error?url={{ currentURL | urlencode }}">let us
```

Comment:
I think GitHub is too specific for techies, so it's better if people submit using our `flag-error` page.